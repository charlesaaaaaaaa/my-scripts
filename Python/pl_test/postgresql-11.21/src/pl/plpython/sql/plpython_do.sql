DO $$ plpy.notice("This is plpython3u.") $$ LANGUAGE plpythonu;

DO $$ plpy.notice("This is plpython2u.") $$ LANGUAGE plpython2u;

DO $$ raise Exception("error test") $$ LANGUAGE plpython3u;
