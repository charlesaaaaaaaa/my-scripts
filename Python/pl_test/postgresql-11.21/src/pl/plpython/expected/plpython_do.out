DO $$ plpy.notice("This is plpython3u.") $$ LANGUAGE plpythonu;
NOTICE:  This is plpython3u.
DO $$ plpy.notice("This is plpython2u.") $$ LANGUAGE plpython2u;
NOTICE:  This is plpython2u.
DO $$ raise Exception("error test") $$ LANGUAGE plpython3u;
ERROR:  Exception: error test
CONTEXT:  Traceback (most recent call last):
  PL/Python anonymous code block, line 1, in <module>
    raise Exception("error test") 
PL/Python anonymous code block
